docs/proposals/device-crd.md

```diff
@@ -292,6 +358,21 @@ spec:
       scale: 1.0
       isSwap: true
       isRegisterSwap: true
+  - propertyName: pressure-enable
+    bluetooth:
+      characteristicUuid: f000aa4204514000b000000000000000
+      dataWrite:
+        ON: [1]
+        OFF: [0]
+  - propertyName: pressure
+    bluetooth:
+      characteristicUuid: f000aa4104514000b000000000000000
```

Comment:
s/characteristicUuid/characteristicUUID/

Reply:
done, thanks for the review :)